osm2streets/src/render.rs

```diff
@@ -307,13 +307,20 @@ impl StreetNetwork {
                 .collect();
             for (a, b) in &intersection.movements {
                 if a != b {
-                    if let Ok(line) = Line::new(road_points[a].0, road_points[b].1) {
-                        pairs.push((
-                            line.to_polyline()
-                                .make_arrow(Distance::meters(0.5), ArrowCap::Triangle)
-                                .to_geojson(Some(&self.gps_bounds)),
-                            make_props(&[]),
-                        ))
+                    if let (Some(pts_a), Some(pts_b)) = (road_points.get(a), road_points.get(b)) {
+                        if let Ok(line) = Line::new(pts_a.0, pts_b.1) {
+                            pairs.push((
+                                line.to_polyline()
+                                    .make_arrow(Distance::meters(0.5), ArrowCap::Triangle)
+                                    .to_geojson(Some(&self.gps_bounds)),
+                                make_props(&[]),
+                            ))
+                        }
+                    } else {
+                        warn!(
+                            "{} has movement ({}, {}) referring to road that is not connected",
+                            &intersection.id, a, b
```

Comment:
Style nit: It should almost never be necessary to `&` or `*` in a `format!` style macro. https://doc.rust-lang.org/std/fmt/trait.Display.html borrows already

Reply:
I swear I've had rust complain that I couldn't move into a `format!` type macro before... I must be misremembering. Maybe I was passing stuff to `make_props` or one of those formatting-adjacent helpers instead.

Good to know, thanks!

Reply:
In case my comment above got lost, it's relevant here: do we need this check now, or can we assume no movements referring to invalid roads're gone?